src/OnboardingSPA/components/HeaderMenuPreview/index.js

```diff
@@ -0,0 +1,68 @@
+import { check, search, Icon } from '@wordpress/icons';
+import { useState, useEffect } from '@wordpress/element';
+
+import { LivePreview } from '../LivePreview';
+
+const HeaderMenuPreview = ( {
+	className = 'live-preview--selectable-card',
+	selected = false,
+	blockGrammer,
+	viewportWidth = 1500,
+	styling = 'large',
+	previewSettings,
+	overlay = false,
+	onClick = false,
+	skeletonLoadingTime = 2500,
+} ) => {
+	const [ loadingParent, setIsLoadingParent ] = useState( true );
+
+	return (
+		<div
+			className={ `${ className }` }
+			onClick={ typeof onClick === 'function' && ( () => {
+				if ( ! loadingParent ) {
+					onClick();
+				}
+			} ) }
+		>
+			<div className={ `${ className }__title-bar` }>
+				<div className={ `${ className }__title-bar__browser` }></div>
```

Comment:
Should we have this? I suppose the CSS is written wrt to this but this is just an empty div.

Reply:
yeah i have reused the CSS used for other steps and without these empty div the tick mark was not appearing at the correct position